packages/bff/package.json

```diff
@@ -19,29 +19,31 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@pagopa/eslint-config": "3.0.0",
+    "@types/adm-zip": "0.5.5",
     "@types/express": "4.17.21",
     "@types/node": "20.14.6",
     "@types/qs": "6.9.15",
     "@types/uuid": "9.0.8",
+    "openapi-zod-client": "1.18.1",
```

Comment:
is this still needed since we moved the generation in another package?

Reply:
🧹  leftover